app/views/StatusView/index.tsx

```diff
@@ -39,24 +41,24 @@ const STATUS: IStatus[] = [
 	},
 	{
 		id: 'offline',
-		name: 'Invisible'
+		name: 'Offline'
 	}
 ];
 
 const styles = StyleSheet.create({
 	inputContainer: {
-		marginTop: 16,
-		marginBottom: 16
-	},
-	inputLeft: {
-		position: 'absolute',
-		left: 12
+		paddingHorizontal: 12,
+		marginTop: 24,
+		marginBottom: 12
 	},
 	inputStyle: {
-		paddingLeft: 48,
 		borderRadius: 0,
 		borderTopWidth: 1,
 		borderBottomWidth: 1
+	},
+	footerComponent: {
+		marginTop: 36,
+		paddingHorizontal: 12
```

Comment:
Confirm, but these horizontal paddings are supposed to be 16.

Reply:
Adjusted.